public/app/ui/components/collections/library/list-result.html

```diff
@@ -1,4 +1,13 @@
 <div class="list-result list-result-library">
     <span class="pull-left" data-bind="component: displayFor('MediaThumbnail', result)"></span>
-    <a href="#" data-bind="attr: { href: detailUrlFor(result) }, component: displayForLabelField(result)"></a>
+    
+    <span class="pull-right right-badge" data-bind="component: displayFor('DateOfPublication', result)"> </span>
```

Comment:
We discussed having this inline with the other publication details, not a fan of having it like this.